AndresConfortiGL/JavaBaseProject
# Added functionality to create a reservation

## Description
Fifth PR - Implemented Reservations to the project.

## Story Type
- New feature :white_check_mark:
- Change request
- Bug fix
- Tech

## How Can This Be Tested?
By pressing the reservation button with a valid parking sport already loaded,
you will be able to input a check in and a check out date, a parking number and a security code.
If you hit continue with the right data, the reservation is created, and sent to the database.
Else, you will be prompted with an error message.

## Evidences
| Nexus S | Pixel 3a |
| ------------- | ------------- |
| ![nexus](https://user-images.githubusercontent.com/109096226/184969177-7c4b0c14-1ce6-40aa-b28c-d7f4e3bf0fe0.gif) | ![pixel](https://user-images.githubusercontent.com/109096226/184970172-35d6dc18-2112-4d16-8ce0-5b450b04026b.gif) |



## Tested on
The project was tested on Android 8.0 / API 26 using the following devices:

- Nexus S
- Pixel 3a

## Package Structure
| Screenshot |
| ------------- |
|![package]![project](https://user-images.githubusercontent.com/109096226/184964519-bdf2bcfe-1f73-4dae-be6e-7743f8446de2.png)|
